mods/ctf/ctf_chat/init.lua

```diff
@@ -11,6 +11,34 @@ function minetest.is_player_name_valid(name)
 	return name:match("^[%a%d_-]+$")
 end
 
+-- Implement coloured PMs by overriding /msg
```

Comment:
Just a small suggestion, but it'll short it a bit, IMHO.
```suggestion
--[[
Implement coloured PMs by overriding /msg
```
And remove all `-` to make it look better.

Reply:
4 lines is too short to warrant comment blocks instead of line comments, IMO

Reply:
Well, IMO it looks weird and doesn't looks good using `--` each line.
But this is up to you: if you want to add it or not, anyway.